.github/workflows/main.yml

```diff
@@ -0,0 +1,60 @@
+---
+name: Main
+
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+    types: [opened, synchronize, reopened]
+
+jobs:
+  test:
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        os:
+          - ubuntu-latest
+          - macos-latest
+          - windows-latest
+        node_version:
+          - 12
+          - 14
+          - 16
+    name: Node ${{ matrix.node_version }} on ${{ matrix.os }}
+    steps:
+      - name: Use LF EOL
+        if: ${{ matrix.os == 'windows-latest' }}
+        run: |
+          git config --global core.autocrlf false
+      - name: Checkout
+        uses: actions/checkout@v2
+      - name: Setup node
+        uses: actions/setup-node@v2
+        with:
+          node-version: ${{ matrix.node_version }}
+      - name: Test
+        if: ${{ matrix.os != 'windows-latest' }}
+        run: |
+          yarn install --frozen-lockfile
+          yarn test
+      - name: Test IE
+        if: ${{ matrix.os == 'windows-latest' }}
+        run: |
+          yarn install --frozen-lockfile
+          yarn test-ie
+
+  codecov:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+      - name: Setup node
+        uses: actions/setup-node@v2
+        with:
+          node-version: 14
+      - name: Test
+        run: |
+          yarn install --frozen-lockfile
+          yarn test
+          yarn codecov
```

Comment:
This is likely going to need a token in secrets. See, e.g. https://github.com/FormidableLabs/trace-deps/blob/main/.github/workflows/ci.yml#L69-L72

I will create and add the secret `CODECOV_TOKEN` in this project.

Reply:
UPDATE -- Looks like codecov has updated features. Let's see if things work without it!